bigtable-test/bigtable-build-helper/pom.xml

```diff
@@ -0,0 +1,161 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <artifactId>bigtable-test</artifactId>
+    <groupId>com.google.cloud.bigtable</groupId>
+    <version>1.14.1-SNAPSHOT</version> <!-- {x-version-update:bigtable-test:current} -->
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.google.cloud.bigtable.test</groupId>
+  <artifactId>bigtable-build-helper</artifactId>
+  <version>1.14.1-SNAPSHOT</version> <!-- {x-version-update:bigtable-test:current} -->
+  <packaging>maven-plugin</packaging>
+  <description>
+    java-bigtable-hbase internal maven extensions.
+    This module is not meant for public, it's primary intention is to help
+    in maintenance of java-bigtable-hbase.
+  </description>
+
+  <properties>
+    <java.version>8</java.version>
+    <maven.version>3.6.3</maven.version>
+    <maven-resolver-api.version>1.6.1</maven-resolver-api.version>
+
+    <!-- treat maven-invoker-plugin tests like integration tests, allowing
+    them to be disabled via skipTests & skipITs -->
+    <skipTests>false</skipTests>
+    <skipITs>${skipTests}</skipITs>
+    <invoker.skip>${skipITs}</invoker.skip>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>org.apache.maven</groupId>
+      <artifactId>maven-core</artifactId>
+      <version>${maven.version}</version>
+      <scope>provided</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.maven</groupId>
+      <artifactId>maven-artifact</artifactId>
+      <version>${maven.version}</version>
+      <scope>provided</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.maven</groupId>
+      <artifactId>maven-model</artifactId>
+      <version>${maven.version}</version>
+      <scope>provided</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.maven</groupId>
+      <artifactId>maven-plugin-api</artifactId>
+      <version>${maven.version}</version>
+      <scope>provided</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.codehaus.plexus</groupId>
+      <artifactId>plexus-utils</artifactId>
+      <version>3.2.1</version>
+      <scope>provided</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.maven.plugin-tools</groupId>
+      <artifactId>maven-plugin-annotations</artifactId>
+      <version>3.5.2</version>
+      <scope>provided</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-api</artifactId>
+      <version>1.7.30</version>
```

Comment:
I think we have this saved as a variable somewhere

Reply:
this is an independent version of slf4j that is provided by maven, so it doesnt relate to other variable. I added a comment